pages/release_notes.rst

```diff
@@ -5,6 +5,32 @@ Release notes
 
 Whats has been changed, and how to update between versions.
 
+*******
+v0.11.0
+*******
+
+See `github project <https://github.com/orgs/nuts-foundation/projects/5>`_ for more details
+
+=======================
+Features / improvements
+=======================
+
+* A version number has been added to the FHIR consent record which is also visible in the consent-store
```

Comment:
Does this mean we always refer to the latest consent version when checking consent? Will there be a specific scenario that makes use of older versions?